### ISSUE TITLE
Extension crashes in Brave

### ISSUE THREAD

**Issue by roshin8** [dessant/buster#165]
* OS name: Windows
* OS version: 10
* Browser name: Brave
* Browser version: Version 1.7.98 Chromium: 81.0.4044.113 (Official Build) (64-bit)

**Extension**

* Extension version: 0.7.0
* User input simulation: Yes
* Client app version: 0.3.0
* Client app installed successfully:  Yes

**Bug description**
The app used to work before but now every time it tries to solve the Audio captcha, it just crashes and no logs are created on the Console. 

There were no Logs created for Browser
Client app Logs:

```
2020/04/27 19:45:04.457638 Starting client (version: 0.3.0)
2020/04/27 19:45:04.457638 Receiving message
2020/04/27 19:45:04.458641 Processing message
2020/04/27 19:45:04.458641 Command: ping
2020/04/27 19:45:04.458641 Sending response
2020/04/27 19:45:04.458641 Receiving message
2020/04/27 19:45:04.486639 Processing message
2020/04/27 19:45:04.486639 Command: moveMouse
2020/04/27 19:45:04.486639 X: 164, Y: 542
2020/04/27 19:45:04.498645 Sending response
2020/04/27 19:45:04.498645 Receiving message
2020/04/27 19:45:04.599992 Processing message
2020/04/27 19:45:04.600575 Command: clickMouse
2020/04/27 19:45:04.621507 Sending response
2020/04/27 19:45:04.621507 Receiving message
2020/04/27 19:45:06.516705 Processing message
2020/04/27 19:45:06.516705 Command: moveMouse
2020/04/27 19:45:06.516705 X: 242, Y: 372
2020/04/27 19:45:06.586699 Sending response
2020/04/27 19:45:06.586699 Receiving message
2020/04/27 19:45:06.776264 Processing message
2020/04/27 19:45:06.776264 Command: clickMouse
2020/04/27 19:45:06.806126 Sending response
2020/04/27 19:45:06.806335 Receiving message
2020/04/27 19:45:10.976613 Closing client

```

**Comment by dessant** [comment]
Did the extension start crashing after a recent Brave release? Does it only crash when the client app is used? Please also check if it crashes with the latest Chrome and Firefox.

**Comment by vinodkunn** [comment]
Hi..
Same problem with me also.
Not only does it crashes, it will also crash some other extension randomly..
The problem started a few days ago, I think. Before that it worked fine..
Also, it works fine with Chrome.

**Comment by dessant** [comment]
@vinodkunn, does brave only crash when user input simulation is enabled?

**Comment by vinodkunn** [comment]
> @vinodkunn, does Brave only crash when user input simulation is enabled?

Tried both ways..
Initially, I did not enable simulation.
Then, when the problem started, I tried user simulation and installed the app.
Still, the same result.

**Comment by PixelHir** [comment]
I'm having the same problem recently, also using brave

**Comment by berkayyildi** [comment]
Brave same problem..

**Comment by dessant** [comment]
Thanks everyone for the details, I'll look into the issue.

**Comment by Hank-Voight** [comment]
The Same here, Brave Crash when i use it with Captcha

Version 1.8.86 Chromium: 81.0.4044.129 (Official Build) (64-bit)

**Comment by dessant** [comment]
An out-of-bounds read was crashing the renderer process in Brave. This bug could lead to remote code execution, so I have reported the vulnerability to the Brave team and they've fixed the issue in Brave Browser 1.8.90.

https://github.com/brave/brave-browser/releases/tag/v1.8.90
